packages/third-parties/agenda/test/agenda-every.integration.spec.ts

```
@@ -66,7 +67,7 @@ describe("Agenda integration", () => {
     afterAll(() => TestContainersMongo.reset());
 
     it("should not have job definitions", () => {
-      const agenda = PlatformTest.injector.get(AgendaService)!;
+      const agenda = PlatformTest.injector.get(Agenda)!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)